components/rich_text_editor/extensions/suggestion/SuggestionList.vue

```diff
@@ -24,6 +25,7 @@
   </div>
 </template>
 
+import type { resolveFocusPosition } from '@tiptap/vue-3';
```

Comment:
does this need to be used?

Reply:
ah my bad, didnt notice this sneak in , vs code autocomplete :(